win/CS/HandBrakeWPF/Views/AudioDefaultsView.xaml

```diff
@@ -375,7 +375,7 @@
                 <Button Content="{x:Static Properties:Resources.AudioDefaultsView_Clear}" cal:Message.Attach="[Event Click] = [Action ClearTracks]" Padding="8,2"/>
 
                 <TextBlock Text="{x:Static Properties:Resources.AudioView_TrackSettingDefaultBehaviour}" Grid.Column="0" Grid.Row="1" Margin="15,0,5,0" HorizontalAlignment="Left"  VerticalAlignment="Center" />
-                <ComboBox Name="audioTrackBeahaviorMode" Grid.Column="1" Grid.Row="2" HorizontalAlignment="Left"
+                <ComboBox Name="audioTrackBehaviorMode" Grid.Column="1" Grid.Row="2" HorizontalAlignment="Left"
```

Comment:
both `behavior` and `behaviour` are used heavily in this file, as sampled here. I'm definitely not enforcing consistency on that term as there were on the order of thousands for both spellings.

Reply:
One of our devs is in the UK. He can't help it. :smile_cat: